ui/app/mainui/SendModalHandler.qml

```diff
@@ -545,6 +556,23 @@ QtObject {
                     return ""
                 }
 
+                property int indexOfTxPathUnderReview: -1
+                readonly property bool reviewingLastTxPath: !!handler.fetchedPathModel && handler.indexOfTxPathUnderReview === handler.fetchedPathModel.rowCount() - 1
```

Comment:
```suggestion
                readonly property bool reviewingLastTxPath: !!handler.fetchedPathModel && handler.indexOfTxPathUnderReview === handler.fetchedPathModel.ModelCount.count - 1
```

This way we're using only bindable properties.


[reference to ModelCount](https://github.com/status-im/status-desktop/blob/7029d56cbf9d2b6cf03f8f81eb9706e58fd83b38/ui/StatusQ/include/StatusQ/modelcount.h)